src/ui/screens/DemocracyScreen.tsx

```diff
@@ -1,67 +1,85 @@
-import * as React from 'react';
-import {SectionList, StyleSheet, View} from 'react-native';
-import type {DeriveProposal, DeriveReferendumExt} from '@polkadot/api-derive/types';
-import {BN_ONE} from '@polkadot/util';
+import React from 'react';
+import {SectionList, StyleSheet, View, RefreshControl} from 'react-native';
 import {useNavigation} from '@react-navigation/native';
 import {StackNavigationProp} from '@react-navigation/stack';
 import {Layout} from '@ui/components/Layout';
 import {EmptyView} from '@ui/components/EmptyView';
 import LoadingView from '@ui/components/LoadingView';
 import {Padder} from '@ui/components/Padder';
-import {ProposalInfo} from '@ui/components/ProposalInfo';
 import SafeView, {noTopEdges} from '@ui/components/SafeView';
 import {SubmitProposal} from '@ui/components/SubmitProposal';
-import {useBestNumber} from 'src/api/hooks/useBestNumber';
-import {useBlockTime} from 'src/api/hooks/useBlockTime';
-import {useDemocracy} from 'src/api/hooks/useDemocracy';
 import {DashboardStackParamList} from '@ui/navigation/navigation';
 import {democracyProposalScreen, referendumScreen} from '@ui/navigation/routeKeys';
-import {Card, Headline, List, Text} from '@ui/library';
+import {Card, Headline, List, Subheading, useTheme} from '@ui/library';
 import globalStyles, {standardPadding} from '@ui/styles';
+import {useDemocracyProposals, DemocracyProposal, DEMOCRACY_PROPOSALS_QUERY} from 'src/api/hooks/useDemocracyProposals';
+import {
+  useDemocracyReferendums,
+  DemocracyReferendum,
+  DEMOCRACY_REFERENDUMS_QUERY,
+} from 'src/api/hooks/useDemocracyReferendum';
+import {useRefetch} from 'src/api/hooks/useRefetch';
+import {ProposalCallInfo} from '@ui/components/ProposalCallInfo';
 
 export function DemocracyScreen() {
-  const {data, isLoading, refetch, isFetching} = useDemocracy();
+  const {data: proposals, loading: proposalsLoading} = useDemocracyProposals();
+  const {data: referendums, loading: referendumsLoading} = useDemocracyReferendums();
```

Comment:
Let's have a single hook for this (`useDemocracy`), you can add to a graphql query any fields you want. Having a single hook will reduce the amount of re-renders and make the code simpler. Also since it will be a single query you case use `refetch` from the result instead of using `useRefetch`